Test/autoRevealDependencies/power.dfy.expect

```diff
@@ -0,0 +1,4 @@
+power.dfy(11,0): Error: a postcondition could not be proved on this return path
+power.dfy(10,31): Related location: this is the postcondition that could not be proved
```

Comment:
It's not clear to me what purpose there is to testing that verification fails in this case. Would it be more valuable to test that it succeeds when the assertion is uncommented, to show that `autoRevealDependencies` hasn't broken the handling of recursive functions?

Reply:
It is a sanity check that the feature is actually working. The program would successfully verify if functions are not made opaque, and so the test would fail.